src/App.tsx

```diff
@@ -1,61 +1,104 @@
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
-
 export const App = () => {
+  const [todos, setTodos] = useState(todosFromServer);
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+  const [titleError, setTitleError] = useState(false);
+  const [userError, setUserError] = useState(false);
+
+  const handleTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+
+    if (titleError) {
+      setTitleError(false);
+    }
+  };
+
+  const handleUserIdChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    setUserId(+event.target.value);
+
+    if (userError) {
+      setUserError(false);
+    }
+  };
+
+  const resetForm = () => {
+    setUserId(0);
+    setTitle('');
+  };
+
+  const handleChange = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    const trimedTitle = title.trim();
+
+    setTitleError(!trimedTitle);
+    setUserError(!trimedTitle);
```

Comment:
Here you should check if user selected or not. You don't need to use title value.